File: app/src/main/java/com/absinthe/anywhere_/viewbuilder/entity/IconPackDialogBuilder.java
package com.absinthe.anywhere_.viewbuilder.entity;

import android.content.Context;
import android.widget.LinearLayout;

import androidx.recyclerview.widget.RecyclerView;

import com.absinthe.anywhere_.viewbuilder.ViewBuilder;

public class IconPackDialogBuilder extends ViewBuilder {

    public RecyclerView rvIconPack;

    public IconPackDialogBuilder(Context context) {
        super(context);
    }

    @Override
    public void init() {
        root = new LinearLayout(mContext);
        root.setLayoutParams(Params.LL.MATCH_MATCH);

        rvIconPack = new RecyclerView(mContext);
        LinearLayout.LayoutParams rvParams = Params.LL.MATCH_WRAP;
        rvParams.setMargins(0, 0, 0, d2p(10));

        rvIconPack.setLayoutParams(rvParams);
        addView(rvIconPack);
    }
}

File: app/src/main/java/com/absinthe/anywhere_/viewbuilder/entity/IconPackDialogBuilder.kt
package com.absinthe.anywhere_.viewbuilder.entity

import android.content.Context
import android.widget.LinearLayout
import androidx.recyclerview.widget.RecyclerView
import com.absinthe.anywhere_.viewbuilder.ViewBuilder

class IconPackDialogBuilder(context: Context?) : ViewBuilder(context!!) {
    @JvmField
    var rvIconPack: RecyclerView = RecyclerView(mContext)

    override fun init() {
        root = LinearLayout(mContext).apply {
            layoutParams = Params.LL.MATCH_MATCH
        }

        rvIconPack = RecyclerView(mContext).apply {
            layoutParams = Params.LL.MATCH_WRAP.apply {
                setMargins(0, 0, 0, 10.dp)
            }
        }

        addView(rvIconPack)
    }
}